src/RollbarLogHandler.php

```diff
@@ -73,55 +74,11 @@ public function log($level, $message, array $context = [])
             return;
         }
 
-        $context = $this->addContext($context);
```

Comment:
I would take the changes from @ArturMoczulski but it doesn't feel right for me as long as he is not agreeing with it.